cypress/e2e/login.cy.js

```diff
@@ -2,10 +2,73 @@
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('https://the-internet.herokuapp.com/login')
   });
 
-  it('', () => {
-    
+  it('should allow to login a user', () => {
+    cy.get('input#username')
```

Comment:
```suggestion
    cy.get('#username')
```
shorter but still valid